src/util/output_channels.ts

```diff
@@ -59,8 +142,12 @@ export async function executeCommandInOutputChannels(command: CliCommand, name:
 
   return new Promise<CliExitData>((resolve) => {
     let stdout = '';
+    let shadowStdout = '';
     let error: string | Error;
+    let shadowError: string | Error;
     let startProcess: ChildProcess;
+    const titleMessage = 'Please provide credentials for image registry.';
+    const selectCredHelper = 'Choose credentials helper';
```

Comment:
```suggestion
    const selectCredHelper = 'Please select the credential helper';
```

Reply:
from this string, I am checking if the deploy command contains the `Choose credentials helper` to show QuickPick to select the QuickPick for Credentials Helper